driver/plan_manager.py

```diff
@@ -1,28 +1,24 @@
 import itertools
-import os
-import os.path
+from pathlib import Path
 import re
 
 from . import returncodes
 
 
-_PLAN_INFO_REGEX = re.compile(r"; cost = (\d+) \((unit cost|general cost)\)\n")
+_PLAN_INFO_REGEX = re.compile(r"; cost = (\d+) \((unit cost|general cost)\)")
```

Comment:
For reference: we used to use `for line in open(...)` which keeps the newline in each line and now use `read_text().splitlines()` which doesn't.